packages/client/src/forms/configuration/configFields/utils.ts

```diff
@@ -139,7 +140,19 @@ export function getFieldDefinition(
 }
 
 export function getContentKey(formField: IFormField) {
-  return formField.label.id
+  if (
+    (formField.type === 'RADIO_GROUP' ||
+      formField.type === 'RADIO_GROUP_WITH_NESTED_FIELDS' ||
+      formField.type === 'SELECT_WITH_OPTIONS' ||
+      formField.type === 'DOCUMENT_UPLOADER_WITH_OPTION') &&
+    !['country', 'countryPermanent', 'nationality'].includes(formField.name)
+  ) {
```

Comment:
We should use the CONSTANTS defined here instead https://github.com/opencrvs/opencrvs-core/blob/fb0985a913ca955f57f3aac87204f83e178f0cb5/packages/client/src/forms/index.ts#L37-L62